lib/sles4sap_publiccloud.pm

```diff
@@ -999,6 +1000,62 @@ sub saphanasr_showAttr_version {
     }
 }
 
+=head2 wait_for_cluster
+
+    Verifies that nodes are online, resources are started and DB is in sync
+
+=cut
+
+sub wait_for_cluster {
+    my ($self, $wait_time, $max_retries) = @_;
+
+    $wait_time //= 10;
+    $max_retries //= 7;
+
+    while ($max_retries > 0) {
+        my $hanasr_output = $self->run_cmd(cmd => 'SAPHanaSR-showAttr --format=script', quiet => 1);
+        my $crm_output = $self->run_cmd(cmd => 'crm_mon -r -R -n -N -1', quiet => 1);
```

Comment:
```
my $crm_output = $self->run_cmd(cmd => $crm_mon_cmd, quiet => 1);
```
`$crm_mon_cmd` is imported from `lib/hacluster.pm`.

Reply:
replaced